packages/xod-project/src/custom-types.js

```diff
@@ -46,3 +51,8 @@ export const listCustomTypeLiteralValidators = () =>
 
 // :: () -> [TypeName]
 export const listCustomTypeNames = () => R.pluck('typeName', typesList);
+
+export const getCustomTypesCompatibility = () =>
```

Comment:
Why is this a function? `BINDABLE_CUSTOM_TYPES_SPECS` is a constant and will never change in runtime.